### PR TITLE
[E2E] Fixing test data for 3.20

### PR DESCRIPTION
**Pull request by yellowee**
Updating test data for 3.20 

## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

File: src/graphql/typePolicies.generated.ts
Comment (yellowee) [inline src/graphql/typePolicies.generated.ts:1]: I am unable to revert this change due to eslint rules - the only change is added line at the end of the file.

Comment (yellowee) [inline src/graphql/typePolicies.generated.ts:1]: I am unable to revert this change due to eslint formatting - the only change is an empty line at the end of the file

**Review by michalina-graczyk — APPROVED** [review]

**Review by poulch — APPROVED** [review]
